tests/acl/test_acl.py

```diff
@@ -701,6 +701,8 @@ def acl_rules(self, request, duthosts, localhost, setup, acl_table, populate_vla
                 with loganalyzer:
                     logger.info("Removing ACL rules")
                     self.teardown_rules(duthost)
+                    wait_until(300, 20, 0, check_msg_in_syslog,
```

Comment:
Same above. 300 seconds look too long to me. We can waste 5 minutes if there is any error happened.